src/compose/ComposeMenu.js

```diff
@@ -83,7 +83,8 @@ class ComposeMenu extends PureComponent<Props> {
       return;
     }
 
-    // $FlowFixMe sketchy falsiness check, because upstream API is unclear
+    // Upstream API is unclear.
+    // $FlowFixMe[sketchy-null-bool]
     const error: string | null = response.error || null;
```

Comment:
Let's try to keep the explanatory text after the $FlowFixMe, so between the $FlowFixMe and the code it's about. That helps keep it clear what text is there to explain the fixme, and what's there to explain something else about the code.

In particular that will help us delete the explanatory text when we delete the fixme, because of either a Flow upgrade or an improvement in the types. Otherwise it'd be easy to pretty often leave it behind because the person deleting the fixme didn't fully understand the comment at that moment, and then there'd be a puzzling comment lingering there to confuse people.

When it's too long for one line, we can use a `/* … */` comment, like in a few of our existing comments and in the docs at https://flow.org/en/docs/errors/ .